docs/tre-admins/setup-instructions/configuring-shared-services.md

```diff
@@ -2,21 +2,81 @@
 
 Complete the configuration of the shared services (Nexus and Gitea) from inside of the TRE environment.
 
-Make sure you run the following command using git bash and set your current directory as C:/AzureTRE
+## Configure Nexus
 
-## Configure Nexus repository
+Before deploying the Nexus shared service, you need to make sure that it will have access to a certificate to configure serving proxies over https with. By default, the Nexus service will serve proxies from `https://nexus-{TRE_ID}.{LOCATION}.cloudapp.azure.com/`, and thus it requires a certificate that validates ownership of this domain to use for SSL.
```

Comment:
```suggestion
Before deploying the Nexus shared service, you need to make sure that it will have access to a certificate to configure serving secure proxies. By default, the Nexus service will serve proxies from `https://nexus-{TRE_ID}.{LOCATION}.cloudapp.azure.com/`, and thus it requires a certificate that validates ownership of this domain to use for SSL.
```